docs/project/changelog.md

```diff
@@ -79,51 +94,46 @@ myst:
   new array with the result values from the iterable appended.
   {pr}`3904`
 
-- {{ Enhancement }} It is now possible to pass environment variables to
-  `loadPyodide` via the `env` argument. `homedir` is deprecated in favor of
-  `{env: {HOME: whatever_directory}}`.
-  {pr}`3870`
+### Deployment
 
 - {{ API }} Changed the name of the default lockfile from `repodata.json` to
   `pyodide-lock.json`
   {pr}`3824`
 
-- {{ Enhancement }} Added `headers` property to `pyodide.http.FetchResponse`.
-  {pr}`2078`
+### Build System
 
-- {{ Breaking }} Changed the `FetchResponse` body getter methods to no longer
-  throw an `OSError` exception for 400 and above response status codes. Added
-  `FetchResponse.raise_for_status` to raise an `OSError` for error status codes.
-  {pr}`3986` {pr}`4053`
+- {{ Update }} The docker image now has node v20 instead of node v14.
+  {pr}`3819`
 
-- {{ Enhancement }} The `setStdin`, `setStdout` and `setStderr` APIs have been
-  improved with extra control and better performance.
-  {pr}`4035`
+- {{ Enhancement }} Adds `check_wasm_magic_number` function to validate `.so`
+  files for WebAssembly (WASM) compatibility.
+  {pr}`4018`
 
 ### Packages
 
-- OpenBLAS has been added and scipy now uses OpenBLAS rather than CLAPACK
-  {pr}`3331`.
 - New packages: sourmash {pr}`3635`, screed {pr}`3635`, bitstring {pr}`3635`,
   deprecation {pr}`3635`, cachetools {pr}`3635`, xyzservices {pr}`3786`,
   simplejson {pr}`3801`, protobuf {pr}`3813`, peewee {pr}`3897`,
   Cartopy {pr}`3909`, pyshp {pr}`3909`, netCDF4 {pr}`3910`, igraph {pr}`3991`,
   CoolProp {pr}`4028`, contourpy {pr}`4102`, awkward-cpp {pr}`4101`.
-- Upgraded libmpfr to 4.2.0 {pr}`3756`.
+
 - Upgraded scipy to 1.11.1 {pr}`3794`, {pr}`3996`
-- Upgraded scikit-image to 0.21 {pr}`3874`
-- Upgraded scikit-learn to 1.3.0 {pr}`3976`
-- Upgraded pyodide-http to 0.2.1
-- Upgraded typing-extensions to 4.7.1 {pr}`4026`
-- Upgraded sourmash to 4.8.3 {pr}`4075`
 
-### CLI
+- OpenBLAS has been added and scipy now uses OpenBLAS rather than CLAPACK
+  {pr}`3331`.
+
+### Pyodide CLI
 
 - {{ Enhancement }} `pyodide build-recipes` now accepts a `--metadata-files`
   option to install `*.whl.metadata` files as specified in
   [PEP 658](https://peps.python.org/pep-0658/).
   {pr}`3981`
 
+### Misc
+
+- {{ Enhancement }} Make it possible to use the @example JSDoc directive.
+  {pr}`4009`
+
```

Comment:
I think this doesn't need a changelog entry (it's a docs infrastructure change).